sdks/python/README.md

```diff
@@ -0,0 +1,17 @@
+# Create virtual enviroment
```

Comment:
could you add a bit more information about these commands?

As a user, when do I need to create virtual env and when do I need to build the python wheel? thx